ports/esp32s2/Makefile

```diff
@@ -173,6 +173,7 @@ SRC_C += \
 	background.c \
 	fatfs_port.c \
 	mphalport.c \
+	pcnt_handler.c \
```

Comment:
Put this in `peripherals`. That is where we usually put helper code that isn't directly tied to the micropython bits.